recipes/opengv/all/conanfile.py

```diff
@@ -146,7 +138,7 @@ def _module_file_rel_path(self):
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "opengv")
         self.cpp_info.set_property("cmake_target_name", "opengv")
-        self.cpp_info.libs = tools.collect_libs(self)
+        self.cpp_info.libs = collect_libs(self)
```

Comment:
I would prefer using explicit library names, but it's fine just updating the tool method for now.